### ISSUE TITLE
Area of a Circle

### ISSUE THREAD

**Issue by abonner1** [learn-co-students/javascript-inheritance-patterns-lab-v-000#92]
I could be misunderstanding what you are asking for, but the area of the circle whose radius you pass in as an argument is not equal to 4 which is the result of `Math.PI * this.radius^2`. Instead, shouldn't it be `Math.PI * this.radius**2`? The `^` is a bitwise operator and not for exponents.

**Comment by andrewjford** [comment]
Yes, the index-test.js line 56 should be: `expect(c.area()).toBe(Math.PI * c.radius**2)`.

**Comment by howardbdev** [comment]
this is odd - the test uses `radius ** 2` , but then the `learn` command wouldn't run for me while it was written that way - so I changed the test and my code to `radius * radius` and it worked .. (`mocha.run()` worked in the browser with `**`)
the solution uses `radius^2`, as mentioned above ..

**Comment by jingruzhang** [comment]
Same here. The test wouldn't run for `radius ** 2`, so I changed to test to `radius * radius`. It worked. I still don't understand what the test `radius ** 2` and the solution code `radius^2` are trying to accomplish...
